src/test/java/org/springframework/data/elasticsearch/core/convert/MappingElasticsearchConverterUnitTests.java

```diff
@@ -211,6 +211,7 @@ public void init() {
 		shotGunAsMap.put("_class", ShotGun.class.getName());
 
 		notificationAsMap = Document.create();
+		notificationAsMap.put("_class", Notification.class.getName());
```

Comment:
hm, that change is not really necessary as line 221 (old) already has this. Although using the class to get the name instead of the string is better.